Archiving/Archiving/Images.xcassets/AppIcon.appiconset/Contents.json

```diff
@@ -2,67 +2,97 @@
   "images" : [
     {
       "idiom" : "iphone",
-      "size" : "29x29",
-      "scale" : "2x"
+      "scale" : "2x",
+      "size" : "20x20"
```

Comment:
Why did this change?

Reply:
I built it and run it and those changes were incorrectly added by me. I will remove them